tests/async/test_async_cluster.py

```diff
@@ -18,7 +18,8 @@ async def test_async_create_cluster_job(async_client: AsyncClient):
     create_res = await async_client.create_cluster_job(
         INPUT_FILE,
         min_cluster_size=3,
-        threshold=0.5,
+        n_neighbors=5,
```

Comment:
I think we should keep the same defaults as UMAP and BERTopic which is I believe 15. Is a reason to go with 5 here? It might produce more local structures vs global ones.

Same for min_cluster_size. BERTopic uses a default of 10.

Reply:
Done! Setting n_neighbors to the BERTopic default of 15 and min_cluster_size to 10. This will give us 2 clusters (sizes 77, 11) and 12 outliers for this dataset.